test/unit/cpu/buildIndexSet.hxx

```diff
@@ -13,7 +13,10 @@
 // for testing...
 //
 
-#include "RAJA/RAJA.hxx"
```

Comment:
This is also a hack; we don't include it, and rely on the file its being included in to include RAJA/RAJA.hxx first.  Also added an include guard that this file was missing.